src/Directive/ArgumentFieldConstraintDirective.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
```

Comment:
Remove this directive, BUT you need to add FIELD_DEFINITION location to existing argument directives, eg StringConstraintDirective